index.js

```diff
@@ -4,5 +4,6 @@
 var BaseTheme = require('ui-base-theme');
 
 module.exports = BaseTheme.extend({
-  name: 'ui-tomato-theme'
+  name: 'ui-tomato-theme',
+  prefix: 'tomato'
```

Comment:
prefix for configuring the component namespace: `[tomato]--ui-button--default.js` for example